code/modules/busy_space/organizations_vr.dm

```diff
@@ -69,7 +69,7 @@
 						"Mari Industrial Complex on Furlou Prime",
 						"Runni Crystal Mines of Keid V")
 
-/datum/lore/organization/unsc
+/datum/lore/organization/mil/unsc
 	name = "United Nations Space Command"
 	short_name = "Navy" //Otherwise they call the ships UNSC UNSC Name
 	desc = "The dedicated military force of SolGov, formed from the remnants of the United Nations, is the might of SolGov. \
```

Comment:
Ye, actually 'mil' is commented out and doesn't show up in the book yet since there's like none. 

Uncomment the bottom of orgs.dm in your PR when you make one.